File: INGEAR.NET/v7.0/NET.LOGIX/Windows/Examples/C Sharp/NETLOGIX.CS.ReadArray/frmNewValue.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace NETLOGIX.CS.ReadArray
{
    public partial class frmNewValue : Form
    {
        public string value = "";
        public frmNewValue()
        {
            InitializeComponent();
        }

        private void frmNewValue_Load(object sender, EventArgs e)
        {
            editValue.Text = value;
        }

        private void btnOK_Click(object sender, EventArgs e)
        {
            value = editValue.Text;
            this.DialogResult = DialogResult.OK;
            this.Close();
        }

        private void btnCancel_Click(object sender, EventArgs e)
        {
            this.DialogResult = DialogResult.Cancel;
            this.Close();
        }


    }
}

File: INGEAR.NET/v7.0/NET.LOGIX/Windows/Examples/Visual Basic/NETLOGIX.VB.ReadArray/frmNewValue.vb
Public Class frmNewValue

    Public value As String

    Private Sub frmNewValue_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        editValue.Text = value
    End Sub

    Private Sub btnOK_Click(sender As Object, e As EventArgs) Handles btnOK.Click
        value = editValue.Text
        Me.DialogResult = DialogResult.OK
        Me.Close()
    End Sub

    Private Sub btnCancel_Click(sender As Object, e As EventArgs) Handles btnCancel.Click
        Me.DialogResult = DialogResult.Cancel
        Me.Close()
    End Sub
End Class